doc/guide/Guide_Euler.ipynb

```diff
@@ -11,42 +11,51 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "\n",
     "### Content\n",
```

Comment:
@NicolasColombi Here's the issue: The very first subheading has the wrong level. The contents sidebar only lists the headings with the same level afterwards. This should fix it:
```suggestion
    "## Content\n",
```

Reply:
@peanutfun Ahh that was it! Thank you Lukas, tomorrow evening I will fix it in all the guides.